### ISSUE TITLE
IP_FILTERLIST_BYPASS_HTTP_AUTH not working ?

### ISSUE THREAD

**Issue by michaelscl** [dutchcoders/transfer.sh#590]
Hi,

Does this feature work well?

With the following configuration, a login window pops up when I try upload file:
```
sudo -E docker run --name transfersh -h transfersh -it --restart unless-stopped \
  -p 8880:8880 \
  -v /etc/timezone:/etc/timezone:ro \
  -v /etc/localtime:/etc/localtime:ro \
  -v ${PWD}/transfersh-basedir:/basedir \
  -v ${PWD}/transfersh-temp:/temp \
  dutchcoders/transfer.sh \
  --provider local \
  --basedir /basedir \
  --temp-path /temp \
  --listener :8880 \
  --purge-days 7 \
  --purge-interval 24 \
  --http-auth-user user --http-auth-pass user 
```
It is OK.

But if I use the following configuration, the login window should pop up too, but it doesn't. The file console.log uploads normally. But my IP address is different from the whitelist.

```
sudo -E docker run --name transfersh -h transfersh -it --restart unless-stopped \
  -p 8880:8880 \
  -v /etc/timezone:/etc/timezone:ro \
  -v /etc/localtime:/etc/localtime:ro \
  -v ${PWD}/transfersh-basedir:/basedir \
  -v ${PWD}/transfersh-temp:/temp \
  dutchcoders/transfer.sh \
  --provider local \
  --basedir /basedir \
  --temp-path /temp \
  --listener :8880 \
  --purge-days 7 \
  --purge-interval 24 \
  --http-auth-user user --http-auth-pass user \
  --http-auth-ip-whitelist 1.2.3.4
```
```
[transfer.sh]2023/12/03 12:14:54 82.202.xx.xx:0 - - [2023-12-03T12:14:54+01:00] "PUT /console.log HTTP/1.0" 200  "https://transfersh.xxxxx.cz/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:120.0) Gecko/20100101 Firefox/120.0"
```

Am I doing something wrong or is it a bug?

best regards

Michaels

**Comment by aspacca** [comment]
> Am I doing something wrong or is it a bug?

what version of the docker image are you using?
a bug disabling http auth was fixed in 1.6.0, and affected version since 1.5.0 and before

also if you use the same browser window to test both configuration, http basic auth header might be cached, and that's the reason why the second time the auth dialog does not pop up

**Comment by michaelscl** [comment]
Great! It works now! Thank you!
